### ISSUE TITLE
High constant memory usage on Synology

### ISSUE THREAD

**Issue by nicke736** [SickRage/SickRage#2450]
I using Synology Diskstation DS2413+ with upgraded memory to 2048MB.

Branch: Master
Commit: dc765c90bc3337ce0d3ed73ffe7424d176a9858d
Database Version: 44.0
OS: Linux-3.10.77-x86_64-with-glibc2.2.5
DSM version: DSM 6.0.2-8451 Update 1
Python version: 2.7.11-15
Python3 version: 3.4.1-4
What you did: Reinstalled sickbeard-custom servral times with new configurations and checked them so they are correct.
What happened: After all shows (18) is added in sickrage memory usage on the Synology stucks on 90-92% and never goes down (2 days and no dropping).
What you expected: I expecting memory usage to be low then searches and processing are inactive. There also no active warnings or errors in the log.
Logs: Attached file with log from latest reinstall, I have censored the apikey with ****.
[Sickrage Debug Log.txt](https://github.com/SickRage/SickRage/files/518039/Sickrage.Debug.Log.txt)

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [ ] Post debug logs, either inline (for smaller logs) or using gist

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by neoatomic** [comment]
Not having any issues on my Syno.

Its probbebly a bad file that is in one of your folders that is coursing the issue.
Sadly the log doesn't tell what file, so best to add the existing folders one by one to see in what folder the file is located.

**Comment by nicke736** [comment]
I started from scratch with a new directory for ongoing TV-shows with Sickrage. I think this helped, it runs much smoother now. Probably like you said with bad file/s. Could also been the subtitle search function because many of the old episodes is missing subtitles, found that hint after a fast search on slow running Sickrage. My archive is pretty big with a lot of files therefore I did a second one instead.
